### PR TITLE
Remove hash and yara table from fuzz harnesses

### PR DESCRIPTION
**Pull request by theopolis**
This should speed up our harnesses. We want to avoid searching the filesystem while fuzzing.

### CODE REVIEW

File: osquery/main/harnesses/fuzz_utils.cpp [hunk 1]
```diff
@@ -17,6 +17,12 @@
 
 namespace osquery {
 
+std::set<std::string> kDisabledFuzzingTables = {
+  "file",
```
Comment (directionless) [inline osquery/main/harnesses/fuzz_utils.cpp:21]: Do you want to disable plist as well? (and there are likely more that are file based)

**Review by directionless — DISMISSED** [review]

File: osquery/main/harnesses/fuzz_utils.cpp (hunk 1 above)
Comment (theopolis) [reply osquery/main/harnesses/fuzz_utils.cpp]: Yeah but these only run on Linux within oss-fuzz.

File: osquery/main/harnesses/fuzz_utils.cpp [hunk 2]
```diff
@@ -17,6 +17,12 @@
 
 namespace osquery {
 
+std::set<std::string> kDisabledFuzzingTables = {
+    "file",
+    "hash",
```
Comment (directionless) [inline osquery/main/harnesses/fuzz_utils.cpp:22]: hash is interesting... Do we fuzz the hash functions elsewhere?

**Review by directionless — APPROVED** [review]

File: osquery/main/harnesses/fuzz_utils.cpp (hunk 2 above)
Comment (theopolis) [reply osquery/main/harnesses/fuzz_utils.cpp:22]: Good question, I was thinking of removing the file enumeration logic too. I'd like to be slightly reactive here and only add alternate paths (fuzz path vs. normal path) selectively.

**Pull request merged**